modules/gitops-installing-argo-rollouts-cli-by-using-linux.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argo-rollouts-cli-on-linux_{context}"]
+= Installing Argo Rollouts CLI on Linux
+
+If you are a Linux user, you can install Argo Rollouts CLI by performing the steps described in this procedure.
+
+.Prerequisites
+
+* You have installed the {OCP} CLI (`oc`)
+
+.Procedure
+
+. Download the latest version of the Argo Rollouts CLI binary, `kubectl-argo-rollouts`, by running the following command:
++
+[source,terminal]
+----
+$ curl -LO https://github.com/argoproj/argo-rollouts/releases/latest/download/kubectl-argo-rollouts-linux-amd64
+----
+
+. Ensure that the `kubectl-argo-rollouts` binary is executable by running the following command:
++
+[source,terminal]
+----
+$ chmod +x ./kubectl-argo-rollouts-linux-amd64
+----
+
+. Move the `kubectl-argo-rollouts` binary to the system path by running the following command:
++
+[source,terminal]
+----
+# mv ./kubectl-argo-rollouts-linux-amd64 /usr/local/bin/kubectl-argo-rollouts
+----
+
+[IMPORTANT]
+====
+Ensure that you have superuser privileges to run this command.
+====
+
+. Verify that the plugin is installed correctly by running the following command and receiving the similar output:
++
+[source,terminal]
+----
+$ oc argo rollouts version
+----
++
+.Example output
+[source,terminal]
+----
+kubectl-argo-rollouts: v1.6.6+737ca89
+  BuildDate: 2024-02-13T15:39:31Z #<1>
+  GitCommit: 737ca89b42e4791e96e05b438c2b8540737a2a1a
+  GitTreeState: clean
+  GoVersion: go1.20.14 #<2>
+  Compiler: gc
+  Platform: linux/amd64 #<3>
+----
+<1> The build date information of the Argo Rollouts binary.
+<2> The version of the Go language used for building the Argo Rollouts binary.
+<3> The platform used for building the Argo Rollouts binary.
```

Comment:
Is it important to point these out?

Reply:
Yes, it adds a bit of additional information for the end-user to refer if the installation is correct or not.